shared/src/business/useCases/trialSessions/getEligibleCasesForTrialSessionInteractor.ts

```diff
@@ -34,13 +37,22 @@ export const getEligibleCasesForTrialSessionInteractor = async (
 
   // Some manually added cases are considered calendared even when the
   // trial session itself is not considered calendared (see issue #3254).
-  let calendaredCases = [];
+  let calendaredCases: (RawCase & TCaseOrder)[] = [];
   if (trialSession.isCalendared === false && trialSession.caseOrder) {
-    calendaredCases = await applicationContext
-      .getUseCases()
-      .getCalendaredCasesForTrialSessionInteractor(applicationContext, {
-        trialSessionId,
-      });
+    calendaredCases = (
+      await applicationContext
+        .getPersistenceGateway()
+        .getCalendaredCasesForTrialSession({
+          applicationContext,
+          trialSessionId,
+        })
+    ).map(aCase => ({
+      ...aCase,
+      docketEntries: aCase.docketEntries.filter(
+        docketEntry =>
+          docketEntry.eventCode === 'PMT' && !docketEntry.isStricken,
+      ),
```

Comment:
Will we still reduce the amount of data passed enough if we don't use a CalendaredCase entity here?

Reply:
🤔 Hmm, you know what? I don't need to do the work of removing these docket entries here because they will be filtered out later in the `EligibleCase` entity. 

I had simply borrowed this code from the interactor that it had been calling, that now removed critical information that this interactor needed.

I'll remove this map and just let the EligibleCase entity do with the case what it wants to do.